app/controllers/clients_controller.rb

```diff
@@ -49,7 +50,7 @@ def destroy
   private
 
   def client_params
-    params.require(:client).permit(:name,:currency_code,address_attributes: [:street_1,:street_2,:city,:state,:pincode,:country_code])
+    params.require(:client).permit(:name,:currency_code,address_attributes: [:street_1,:street_2,:city,:state,:pincode,:country_code], contact_detail_ids: [], contact_details_attributes: [:email, :first_name, :last_name, :phone, :mobile])
```

Comment:
Limit line to 80 characters https://github.com/bbatsov/ruby-style-guide#80-character-limits